Pointnav_DDPPO_baseline.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM fairembodied/habitat-challenge:testing_2020_habitat_base_docker
+
+RUN /bin/bash -c ". activate habitat; pip install ifcfg torch tensorboard"
```

Comment:
@Skylion007 we need to make sure to get a CUDA 10.1 version of torch here.  Looks like the AWS machines don't support CUDA 10.2 (which is pulled by default)